ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -799,15 +797,6 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
         }
     }
 
-    private func updateAppearance() {
-        updateBackgroundColor()
-
-        guard let presentationController = (presentationController as? DrawerPresentationController) else {
-            return
-        }
-        presentationController.updateApperance()
```

Comment:
Does the presentation controller still get updates?

Reply:
The updates in this function are to the shadowView, the dimmingView which are already handled separately.